app/models/katello/glue/candlepin/owner.rb

```diff
@@ -30,7 +30,6 @@ def candlepin_owner_exists?
       def owner_details
         @owner_details ||= find_owner
         @owner_details['virt_who'] ||= self.subscriptions.using_virt_who.any?
-
```

Comment:
Any reason for deleting this line, or just cleaning up things?

Reply:
oh I think that was just left over from a log while debugging, I'll put it back in!